stdlib/public/core/UnsafePointer.swift.gyb

```diff
@@ -429,6 +429,11 @@ public struct ${Self}<Pointee>
     return UnsafeMutablePointer(rawPtr)
   }
 
+  @available(*, unavailable, message: "Swift currently only supports freeing entire heap blocks, use deallocate() instead")
```

Comment:
I see that, but using `unavailable` causes source breakage of Swift 4 code, not just code that is migrated to Swift 5.